cmd/notary/tuf.go

```diff
@@ -692,6 +692,15 @@ func getRemoteTrustServer(config *viper.Viper) string {
 	return defaultServerURL
 }
 
+func getUseNative(config *viper.Viper) bool {
+	conf := config.Get("useNative")
```

Comment:
I thought of doing this just in case a user enters an invalid boolean, your call- I'm fine with either